### PR TITLE
Xenoarch Machine Update

### PR DESCRIPTION
**Pull request by jjpark-kb**
## About The Pull Request
The code for xenoarch was made when I decided I didn't want to use process-- yea, I'm changing this to process and cooldowns. There is also some buffs-- both stock parts affect the speed and the default speed is now 10, going down to 4 at max. Additionally, I brought back the previous two iterations of the researcher-- you can now either get the lavaland loot (the current), get the anomalous crystal (the previous), or bepis tech (the previous previous).

## How This Contributes To The Skyrat Roleplay Experience
xenoarch should see more use

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image
image
image

</details>

## Changelog

:cl:
balance: xenoarch machines will now default take 10 seconds instead of 20 to do their process
add: the xenoarch researcher can now create lavaland loot, anomalous crystals, or bepis tech
add: the xenoarch machines will now be affected by both their stock parts
fix: you can now wrench, screwdriver, and crowbar the xenoarch machines
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/xenoarch/code/modules/research/xenoarch/xenoarch_machine.dm [hunk 1]
```diff
@@ -1,138 +1,164 @@
...
+		/obj/item/xenoarch/useless_relic/magnified = 10,
+		/obj/item/xenoarch/broken_item = 10,
+	)
 
 /obj/machinery/xenoarch/researcher/examine(mob/user)
 	. = ..()
-	. += span_notice("[current_research]/150 research points. Research more xenoarchaeological items.")
+
+	. += span_notice("<br>[current_research]/[max_research] research available.")
+	. += span_notice("L-Click to insert items or take out all the strange rocks. R-Click to use research points.")
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/xenoarch/code/modules/research/xenoarch/xenoarch_machine.dm:87]: Could turn these into tooltips at the top of the screen?

Comment (jjpark-kb) [reply modular_skyrat/modules/xenoarch/code/modules/research/xenoarch/xenoarch_machine.dm:87]: I... probably could, yes.

**Comment by Gandalf2k15** [comment]
did ti work?

**Comment by jjpark-kb** [comment]
> did ti work?

NO ;-;

**Comment by Gandalf2k15** [comment]
eh then might as well leave it be

**Comment by jjpark-kb** [comment]
back to before the contextual tips

**Pull request merged**